target/ppc/translate/vmx-impl.c.inc

```diff
@@ -1238,6 +1238,41 @@ GEN_VXFORM_DUAL(vspltish, PPC_ALTIVEC, PPC_NONE,
 GEN_VXFORM_DUAL(vspltisw, PPC_ALTIVEC, PPC_NONE,
                 vinsertw, PPC_NONE, PPC2_ISA300);
 
+static bool do_vextdx(DisasContext *ctx, arg_VA *a, int size, bool right,
+                      void (*gen_helper)(TCGv_ptr, TCGv_ptr, TCGv_ptr, TCGv))
+{
+    TCGv rc;
+    TCGLabel *l0;
+
+    REQUIRE_INSNS_FLAGS2(ctx, ISA310);
+    REQUIRE_VECTOR(ctx);
+
+    rc = tcg_temp_local_new();
```

Comment:
AFAIK, we don't have something like `gen_invalid` to call from the helper, so we'd need to return something to indicate the result and branch accordingly. It would spare us the local temp, but we'd have an expensive helper call for something that can be solved in TCG.

Reply:
gen_invalid? The branch is just skipping the result calculation when idx is too large, isn't it?